tests/test_cms_config.py

```diff
@@ -123,10 +162,66 @@ def test_versionables_list_created(self):
             djangocms_versioning_enabled=True,
             versioning=[poll_versionable, blog_versionable]
         )
-        extension.handle_versioning_setting(cms_config)
+        extension.configure_app(cms_config)
         self.assertListEqual(
             extension.versionables, [poll_versionable, blog_versionable])
 
+    def test_context_dict_created(self):
+        """Test a dict is populated from the versioning_add_to_confirmation_context
+        config
+        """
+        extension = VersioningCMSExtension()
+
+        def unpublish_context1(request, version, *args, **kwargs):
+            return "Every time you unpublish something you should kiss a cat"
```

Comment:
Do we need to return if you are not testing it? `pass` should do the job. Although the message does spice up reading the test.

Reply:
These messages are asserted later in the test, so I do actually need them... :cat2: :mouse2:

Reply:
Hmmm, although actually not in that test :D I guess I could put `pass`, but I think I prefer to return some sort of string as that's what a real function would do (but I can make the message less spicy if you think could be a problem ;)).

Reply:
@chaosk Any opinions on this?

Reply:
I don't have a preference here, you can keep it as it is